### ISSUE TITLE
Potential issue with region ap-east-1

### ISSUE THREAD

**Issue by ArtificialParLjungstrom** [awslabs/aws-instance-scheduler#131]
I have set up Instance Scheduler in region eu-west-1 in account A and  added scheduling and remote permissions for it to start and stop EC2 instances in regions eu-west-1, us-west-2, ap-southeast-1 and ap-east-1 in account B. The scheduling works fine for regions eu-west-1, us-west-2 and ap-southeast-1 but no actions are performed in region ap-east-1.

Cloudwatch log output shows that the scheduler seems to stop when trying to describe instances in the ap-east-1 region:
2019-10-17 - 08:00:32.634 - INFO : Handler SchedulerRequestHandler scheduling request for service(s) ec2, account(s) B, region(s) ap-east-1 at 2019-10-17 08:00:32.634154
2019-10-17 - 08:00:33.881 - INFO : Running EC2 scheduler for account B using role arn:aws:iam::B:role/Instance-Scheduler-EC2SchedulerCrossAccount-B in region(s) ap-east-1
2019-10-17 - 08:00:33.567 - INFO : Fetching ec2 instances for account B in region ap-east-1
Cloudtrail output from account B doesn't show any API activity for the region at the scheduled times.

I have used the default templates, at https://docs.aws.amazon.com/solutions/latest/instance-scheduler/templates.html, so the remote EC2 permissions are set with a wildcard to cover all regions in account  B.

**Comment by FrankFioretti** [comment]
ap-east-1 is an opt-in region by default, so all api actions to it will fail unless the region has been opted in to in a given account.

**Comment by ArtificialParLjungstrom** [comment]
Yeah, that was my suspicion as well but the problem persists even after the region has been activated for both accounts. I even recreated the scheduler setup after making sure the region was activated.
